README.md

```diff
@@ -10,22 +10,18 @@
 
 
    - Day1 
-	   - 9-11: 		Course introduction
-	   - 11-12:30: 		UNIX
-	   - 12:30-13:30: 	Lunch break
-	   - 13:30-18: 		UNIX
-
+	   - 8-10: 		Course introduction
```

Comment:
Are we sure we can start so early?

Reply:
Wait for Andrea's confirmation of classroom booking!!